g9frontend/src/pages/UserPage/UserPage.tsx

```diff
@@ -40,8 +41,8 @@ export const UserPage: React.FC = () => {
                         right: '3%',
                         fontFamily: 'Inter, sans-serif',
                         fontSize: '20px'
-                    }}>
-                        Log in
+                    }} to="/" component={Link} color="inherit" sx={{mr: 5}}>
+                        Home Page
```

Comment:
Please add this icon to the button: `import PersonOutlineIcon from '@mui/icons-material/PersonOutline';`, like in the Figma design.